src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -114,10 +114,13 @@ trait WorkspaceApiService extends FireCloudRequestBuilding with FireCloudDirecti
                   post {
```

Comment:
I only did this for flexibleImportEntities. Should I bother with `importEntities`?

Reply:
Terra UI calls `importEntities` instead of `flexibleImportEntities` if the end user checks the _Create participant, sample, and pair associations_ box. I assume we also want to support blanks-as-removals for that use case.

Reply:
Terra UI calls `importEntities` instead of `flexibleImportEntities` if the end user checks the _Create participant, sample, and pair associations_ box. I assume we also want to support blanks-as-removals for that use case.